docs/src/reconciling-local-changes.md

```diff
@@ -0,0 +1,230 @@
+# Reconciling changes made outside of TPA
+
+In general, if a cluster was created using TPA it is advisable to manage
```

Comment:
This is very soft and fuzzy and maybe should be more explicit?

Any changes made to a TPA created cluster that are not performed by changing the TPA configuration will not be saved. This means that your cluster will have changes that the TPA configuration won't  be able to recreate. 

This page shows how configuration is managed with TPA and the preferred ways to make configuration changes. We then look at strategies to make, and reconcile, the results of making manual changes to the cluster.